cfgov/v1/models/organisms.py

```diff
@@ -137,3 +138,18 @@ class Meta:
 
     class Media:
         js = ("expandable-group.js",)
+
+
+class ItemIntroducation(blocks.StructBlock):
```

Comment:
At some point can we break apart organisms.py and molecules.py so there's one python file > one template per atomic element?